igvm/vm.py

```diff
@@ -1106,6 +1106,34 @@ def aws_sync(self) -> dict:
 
         return sync_values
 
+    def is_aws_image_golden(self) -> bool:
+        """Return whether the VM images is golden (created by us) or not"""
+
+        response = self.ec2c.describe_images(
+            ImageIds=[
+                self.dataset_obj['aws_image_id'],
+            ]
+        )
+
+        tags = response['Images'][0].get('Tags')
+
+        if not tags:
+            log.warning(
```

Comment:
This is not what I meant. We are not interested if there are no tags. We are interested in whether the image was found and returned from the API